packages/synchro-charts/src/components/charts/sc-webgl-base-chart/sc-webgl-base-chart.tsx

```diff
@@ -829,6 +831,9 @@ export class ScWebglBaseChart {
     const { marginLeft, marginTop, marginRight, marginBottom } = chartSizeConfig;
 
     const hasError = this.dataStreams.some(({ error }) => error != null);
+    const hasSupportedDataTypes = this.dataStreams.every(
```

Comment:
we can use visualizedDataStreams() here to handle the alarm check, not a huge performance diff

Reply:
nvm I'm wrong :) this is fine